packages/storybook/package.json

```diff
@@ -6,8 +6,10 @@
   "license": "Apache-2.0",
   "main": "index.js",
   "scripts": {
-    "build:storybook": "yarn generate:versions && storybook build -o dist",
+    "build:storybook": "yarn install:browsers && yarn generate:versions && storybook build -o dist && yarn extract:llms",
+    "extract:llms": "../../node_modules/.bin/storybook-llms-extractor --config llms.config.js && node scripts/postprocess-llms.js",
     "generate:versions": "node scripts/generate-version-list.js 'assets/ods-versions.json'",
+    "install:browsers": "../../node_modules/.bin/playwright install --with-deps chromium",
```

Comment:
shouldn't be needed, playwright should be able to be installed and run with the basic install setup